<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>warnningwidget</class>
 <widget class="QMainWindow" name="warnningwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>191</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>191</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>191</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>提示</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: palette(window);  /* 与界面背景色一致 */
    border: none;                       /* 无边框 */
}</string>
      </property>
      <property name="text">
       <string>文件不存在！</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
